### ISSUE TITLE
--nofreeze-stdlib fails to include the stdlib

### ISSUE THREAD

**Issue by mathrick** [kayhayen/Nuitka#61]
*  Nuitka version: `0.5.29rc4`@`60b7d44` on Mint 18 (essentially the same as Ubuntu 16.04)
*  Installed via `pip install git+https://github.com/kayhayen/Nuitka`
*  Repro:
```python
import os
```

Running `nuitka --nofreeze-stdlib --python-flag=no_site --standalone foo.py` yields the following:

```
$ ./foo.dist/foo.exe 
Traceback (most recent call last):
  File "/tmp/foo.dist/foo.py", line 1, in <module>
ImportError: No module named os
```

**Comment by kayhayen** [comment]
I can reproduce this. Seems nothing outside foo.py will get compiled unless one also specifies "--recurse-stdlib", which does then. I guess in standalone mode they should be joined. As freezing the stdlib is only relevant there, this kind of shows the whole option is bogus. I will remove it. I think you are not the first to fall for it.

**Comment by kayhayen** [comment]
Fixed in 0.5.29
